src/lib/cli/progress.js

```diff
@@ -0,0 +1,83 @@
+// @flow
+
+/**
+ * External dependencies
+ */
+import chalk from 'chalk';
+import { stdout as progressLog } from 'single-line-log';
+
+// Progress spinner
+const sprite = {
```

Comment:
This `progress` component is useful. I think it makes sense to use the same component for sync progress too (in a future PR)

Reply:
Yup, definitely!